components/raftstore/src/coprocessor/mod.rs

```diff
@@ -247,11 +246,20 @@ impl ObserveHandle {
 pub struct CmdObserveInfo {
     pub cdc_id: ObserveHandle,
     pub rts_id: ObserveHandle,
+    pub pitr_id: ObserveHandle,
 }
```

Comment:
What about an enum? Also we can replace `ObserverType` with `CmdObserveInfo`.

Reply:
It seems that `CmdObserverInfo` may contain multi types of observers (e.g. when the cluster enabled both `resolved-ts` and `cdc`, both of `cdc_id` and `rts_id` would be non-default.) So I'm afraid we cannot make it an `enum` trivially. 🤔